js/gated-content/src/components/AddToFavorite.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    :class="{ active: isActive, 'add-to-favorite': true }"
```

Comment:
Shouldn't we move the 'add-to-favorite' a line below?

Reply:
Fixed